src/test/Figure/Population.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
```

Comment:
There's been a bit of discussions regarding possibly namespacing the definitions under the `/test/` path by GitHUb username. I'll need to check if we made any decision about requiring that or not. I'll get back to that if necessary.